### PR TITLE
Repl Enhancements

### PR DESCRIPTION
**Pull request by AnEmortalKid**
* Added `--run` option for evaluating dogescript:

```
$ dogescript hello_world.djs --run
[dogescript@2.4.0-pre]
Hello World!
```

* Deprecated true-doge with a warning at the repl level
```
$ dogescript hello_world.djs --true-doge
[dogescript@2.4.0-pre]
!-- WARNING: true-doge mode is deprecated --!
console.log('Hello World!');
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ![dogescript](doge.gif)
 
-## dogescript 
+## dogescript
```
Comment (vpzomtrrfrt) [inline README.md]: Not sure exactly what this change is, but it doesn't seem relevant to this PR

File: bin/dogescript.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ var path     = require('path');
 var util     = require('util');
 var stream   = require('stream');
 var repl     = require('repl');
-var argv     = require('optimist').usage('Usage: dogescript <file>').argv;
+var argv     = require('optimist').usage('Usage: dogescript <file> [options]').argv;
```
Comment (vpzomtrrfrt) [inline bin/dogescript.js]: Options traditionally go before the file, as far as I can tell.

Also, I don't know where/if this line shows up, I just get the REPL every time I try

File: bin/dogescript.js [hunk 3]
```diff
@@ -16,14 +16,34 @@ process.stdout.write("[dogescript@"+pjson.version+"]\n");
...
-        if (argv['true-doge']) var lines = script.split(/ {3,}|\r?\n/);
-        else var lines = script.split(/\r?\n/);
+        if (argv['true-doge'])
+        {
+          process.stdout.write("!-- WARNING: true-doge mode is deprecated --!\n");
+          var lines = script.split(/ {3,}|\r?\n/);
+        }
+        else
+        {
+          var lines = script.split(/\r?\n/);
```
Comment (vpzomtrrfrt) [inline bin/dogescript.js]: Putting the declaration outside the if would look nicer, since in most languages it's required to be that way

File: bin/dogescript.js [hunk 4]
```diff
@@ -16,14 +16,34 @@ process.stdout.write("[dogescript@"+pjson.version+"]\n");
...
+        }
         var output = '';
 
         for (var i = 0; i < lines.length; i++) {
             output += parser(lines[i]);
         }
 
+        // allow run mode
+        if(argv['run'])
```
Comment (vpzomtrrfrt) [inline bin/dogescript.js]: using dot notation would make this more consistent with the `argv.beautify` line below

File: bin/dogescript.js (hunk 2 above)
Comment (AnEmortalKid) [reply bin/dogescript.js]: i think if you don't type anything you just get the repl... i can probably add a `-h` in a following pull

File: README.md (hunk 1 above)
Comment (AnEmortalKid) [reply README.md]: might be CRLF -> LF. I'll check with a .gitattributes :)

File: bin/dogescript.js (hunk 2 above)
Comment (AnEmortalKid) [reply bin/dogescript.js]: >Options traditionally go before the file, as far as I can tell. 

I think that is the case, however the repl already was defined to have the options like that... so i just kept the order and updated the doc.

File: README.md (hunk 1 above)
Comment (AnEmortalKid) [reply README.md]: Ah, it seems there was a ` ` character in there (not a line char)

```
## dogescript ]
```
